src/library/basesqltablemodel.h

```diff
@@ -139,6 +139,23 @@ class BaseSqlTableModel : public QAbstractTableModel, public TrackModel {
         }
         int m_column;
         Qt::SortOrder m_order;
+        
+        friend QTextStream& operator >> (QTextStream &in, SortColumn &sc) {
```

Comment:
@jmigual: Would you mind to put this rule of thump as comment into

 mixxx/src/library/dao/settingsdao.h

Thank you.